com.gitlab.inendi.Inspector.yaml

```diff
@@ -0,0 +1,23 @@
+app-id: com.gitlab.inendi.Inspector
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: inspector_launcher.sh
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=host
+  - --filesystem=/opt/inendi
+  - --filesystem=~/.inendi
```

Comment:
I think what tou want is `persist`. otherwise this is redundant with the overly broad `host` permission.

Reply:
My bad